mon-pix/app/templates/components/certification-joiner.hbs

```diff
@@ -4,10 +4,21 @@
 
 <section class="certification-joiner">
   <h1 class="certification-joiner__title">{{t "pages.certification-joiner.first-title"}}</h1>
-    <form autocomplete="off">
+    <form autocomplete="off" {{on "submit" this.attemptNext}}>
         <div class="certification-joiner__row">
           <label class="certification-joiner__label" for="certificationJoinerSessionId">{{t "pages.certification-joiner.form.fields.session-number"}}</label>
-          <Input {{autofocus}} @type="text" @size="6" @value={{this.sessionId}} id="certificationJoinerSessionId" />
+          <Input
+            {{autofocus}}
+            id="certificationJoinerSessionId"
+            pattern={{this.sessionPattern}}
+            @type="text"
+            @size="6"
+            @value={{mut this.sessionId}}
+            {{on "input" this.checkIsANumber}}
+            @required="true" />
+          {{#if this.sessionIdIsNotANumberError}}
```

Comment:
pour ce serait pas un flag ici {{#if this.sessionIdNumberError}} et on cale le texte dans le template direct ?